### PR TITLE
View post page

### LINKED ISSUE #53: Create view post page

### PR DESCRIPTION
**Pull request by ZhinAbubakr**

### CODE REVIEW

**Comment by ZhinAbubakr** [comment]
image

**Review by murtaja1 — APPROVED** [review]

File: src/components/ViewPost/ViewPost.jsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+function ViewPost() {
+  const classes = useStyles();
+
+  return (
+    <>
+        <Container className={classes.container}>
+            <Paper className={classes.paper} >
+                <ThemeProvider theme={theme}>
+                    <Typography variant="h4" className={classes.headerTypo}>
+                        Item Lost Information:
```
Comment (MuhannedNoman) [inline src/components/ViewPost/ViewPost.jsx:24]: Any text should be using i18next for easier translation along the road!

File: src/components/ViewPost/ViewPost.jsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                    </Typography>
+                    <Grid container>
+                        <Grid xs={16} sm={8} md={3}>
+                            <CardMedia
+                                className={classes.media}
+                                image="https://miro.medium.com/max/1200/1*mk1-6aYaf_Bes1E3Imhc0A.jpeg"
+                                title=""
```
Comment (MuhannedNoman) [inline src/components/ViewPost/ViewPost.jsx:31]: Why are you passing an empty prop?

File: src/components/ViewPost/ViewPost.jsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                            />
+                        </Grid>
+                        <Grid item xs={12} sm={6} md={8}>
+                            <ListItem>
+                                <Container maxWidth="lg" className={classes.container2}>
+                                    <Typography variant="body1" color="textSecondary">
+                                        Item Name:
+                                    </Typography>
+                                    <Typography variant="body1">
```
Comment (MuhannedNoman) [inline src/components/ViewPost/ViewPost.jsx:40]: Not sure about this butt `body1` is not a good name, if possible make it meaningful!

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]

**Comment by MuhannedNoman** [comment]
Hey @ZhinAbubakr , outstanding work here! I have left you some comments take a look and let me know what you think!

File: src/components/ViewPost/ViewPost.jsx (hunk 3 above)
Comment (murtaja1) [reply src/components/ViewPost/ViewPost.jsx:40]: this is a variant from material UI.

**Pull request merged**